src/test/java/core/basesyntax/service/ParserImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserImplTest {
+    private static Parser parser;
+    private static List<String> lines;
```

Comment:
don't make tested class and test data static because in such way they will be shared between tests and tests may affect each other. fix in all tests